src/sentry/eventstream/base.py

```diff
@@ -71,20 +72,24 @@ def _dispatch_post_process_group_task(
         primary_hash: Optional[str],
         skip_consume: bool = False,
         group_states: Optional[GroupStates] = None,
+        group_category: Optional[GroupCategory] = GroupCategory.ERROR,
     ) -> None:
         if skip_consume:
             logger.info("post_process.skip.raw_event", extra={"event_id": event_id})
         else:
             cache_key = cache_key_for_event({"project": project_id, "event_id": event_id})
 
-            post_process_group.delay(
-                is_new=is_new,
-                is_regression=is_regression,
-                is_new_group_environment=is_new_group_environment,
-                primary_hash=primary_hash,
-                cache_key=cache_key,
-                group_id=group_id,
-                group_states=group_states,
+            post_process_group.apply_async(
+                kwargs={
+                    "is_new": is_new,
+                    "is_regression": is_regression,
+                    "is_new_group_environment": is_new_group_environment,
+                    "primary_hash": primary_hash,
+                    "cache_key": cache_key,
+                    "group_id": group_id,
+                    "group_states": group_states,
+                },
+                queue=get_post_process_queue(group_category),
```

Comment:
Small thing about apply_async, I think you gotta put the `queue` kwarg within `kwargs` since `post_process_group` is an instrumented_task and has the `queue` param pinned already. 

There seems to already be discussion on creating a separate task altogether for different categories, but this change will fix some of the failing tests.

Reply:
I think celery overrides the `kwargs` argument to pass to the function and uses as `options` for the same effect